app/assets/javascripts/components/autocomplete.js

```diff
@@ -133,5 +135,55 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     $select.parentNode.removeChild($select)
   }
 
+  Autocomplete.prototype.initAutoCompleteSearchTopics = function () {
+    var millerColumns = document.querySelector('miller-columns')
+    var topics = millerColumns.taxonomy.flattenedTopics
+
+    if (!topics) {
+      return
+    }
+
+    new window.accessibleAutocomplete({ // eslint-disable-line no-new, new-cap
+      id: 'topics-autocomplete',
+      name: 'topics-autocomplete',
+      element: this.$module,
+      minLength: 3,
+      autoselect: false,
+      source: function (query, syncResults) {
+        var results = topics
+
+        syncResults(query
+          ? results.filter(function (result) {
+            var indexOf = result.topicName.toLowerCase().indexOf(query.toLowerCase())
+            var resultContainsQuery = indexOf !== -1
+            if (resultContainsQuery) {
+              // Strip existing <mark> tags
+              result.highlightedTopicName = result.topicName.replace(/<\/?mark>/gm, '')
+              // Wrap query in <mark> tags
+              var queryRegex = new RegExp('(' + query + ')', 'ig')
+              result.highlightedTopicName = result.topicName.replace(queryRegex, '<mark>$1</mark>')
+            }
+            return resultContainsQuery
+          }) : []
+        )
+      },
+      templates: {
+        inputValue: function (result) {
+          return result && result.topicName && ''
+        },
+        suggestion: function (result) {
+          var suggestions = result.topicNames
+          suggestions[suggestions.length - 1] = result.highlightedTopicName
+          return result && suggestions.join(' › ')
```

Comment:
Also a bit confused by the result && here

Reply:
I'm using conditional statement in boolean algebra consistent with other custom autocompletes and examples. We can expand it as a conditional expression, but I'll have to update the rest of custom inits too.

Essentially if we have a result we return the suggestions as a string of breadcrumbs items.

Reply:
but won't that already error as you've tried to call `.topicNames` on result earlier?

Reply:
Yep, that's true, I forgot we expanded the logic within this function.